docs/sources/next/javascript-api/jslib/http-instrumentation-tempo/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'HTTP instrumentation for Tempo'
+menuTitle: http-instrumentation-tempo
+description: 'k6 Tempo instrumentation API'
+weight: 04
+---
+
+# HTTP instrumentation for Tempo
+
+With this jslib, you can _instrument_ HTTP requests so that they emit traces as the test runs. Use it to include a tracing context in HTTP requests so that tracing backends (such as [Grafana Tempo](https://grafana.com/oss/tempo/)) can incorporate their results.
```

Comment:
I would link to our k6 x Tempo [docs](https://grafana.com/docs/grafana-cloud/testing/k6/analyze-results/integration-with-grafana-cloud-traces/?pg=blog&plcmt=body-txt) here. Also, the last sentence doesn't really make sense to me. What does "tracing backends [...] can incorporate their results" mean?

Reply:
```suggestion
With this jslib, you can _instrument_ HTTP requests so that they emit traces as the test runs. Use it to include a tracing context in HTTP requests so that tracing backends such as [Grafana Tempo](https://grafana.com/docs/grafana-cloud/testing/k6/analyze-results/integration-with-grafana-cloud-traces/?pg=blog&plcmt=body-txt).
```

Reply:
@Blinkuu I think if this is okay - I will just merge this PR and we can always edit the docs after this.

Reply:
```suggestion
With this jslib, you can _instrument_ HTTP requests so that they emit traces as the test runs. Use it to include a tracing context in HTTP requests, which can then be used by a tracing backend such as [Grafana Tempo](https://grafana.com/docs/grafana-cloud/testing/k6/analyze-results/integration-with-grafana-cloud-traces/).
```

How about this?